crates/uv-resolver/src/resolver/mod.rs

```diff
@@ -1402,6 +1403,20 @@ impl<InstalledPackages: InstalledPackagesProvider> ResolverState<InstalledPackag
                     return Ok(None);
                 };
 
+                // Avoid prefetching source distributions with unbounded lower-bound ranges. This
+                // often leads to failed attempts to build legacy versions of packages that are
+                // incompatible with modern build tools.
```

Comment:
Love it :)

Reply:
Nice, simple solution to a simple problem. Let met confirm it works :)